doc/DOCUMENTATION.org

```diff
@@ -2078,8 +2078,8 @@ Basic =ido= operations can be done with ~Ctrl~ key:
 
 | Key Binding        | Description                                       |
 |--------------------+---------------------------------------------------|
-| ~C-<return>~       | open a =dired buffer=                             |
-| ~M-<return>~       | open a =dired buffer= in terminal                 |
+| ~C-RET~            | open a =dired buffer=                             |
+| ~M-RET~            | open a =dired buffer= in terminal                 |
```

Comment:
Wait a sec. These are supposed to be defined in Ido (`ido-mode`), so you need to run `M-x ido-find-file` and from that prompt test the key bindings. When I checked it `C-RET` actually did work, I was only talking about `M-RET`.